deepdrivemd/sim/openmm_stream/config.py

```diff
@@ -0,0 +1,57 @@
+from enum import Enum
+from pathlib import Path
+from typing import Optional, List
+from pydantic import root_validator
+from deepdrivemd.config import MolecularDynamicsTaskConfig
+
+
+class OpenMMConfig(MolecularDynamicsTaskConfig):
```

Comment:
Can subclass non-streaming OpenMMConfig

Reply:
I would rather keep configs independent. Cleaned and removed some unused parameters.